.github/workflows/publish.yml

```diff
@@ -10,8 +10,8 @@ jobs:
     site-build:
         name: Build & publish site
         runs-on: ubuntu-latest
-        # Run the job if manually triggered or if the commit message includes '#publish' & the check suite has passed
-        if: github.event_name == 'workflow_dispatch' || contains(github.event.head_commit.message, '#publish')
+        # Run the job if manually triggered or if the commit message includes '#publish' & the check suite has passed or if the commit message includes 'docs' and the check suite has passed
+        if: github.event_name == 'workflow_dispatch' || contains(github.event.head_commit.message, '#publish') || contains(github.event.head_commit.message, 'docs')
```

Comment:
this will publish the docs site if the `docs` commit type is included in the merge message